.github/workflows/verdi.sh

```diff
@@ -34,3 +34,6 @@ while true; do
     fi
 
 done
+
+$VERDI devel check-load-time
+$VERDI devel check-undesired-imports
```

Comment:
Won't this just run `which verdi ...`? Will this actually call the verdi commands?

Reply:
`which verdi` will just resolve to the full path of the verdi binary, and it should then just be executed normally. Note that we don't really need to call `check-load-time` because that is exactly already done in the bash code in this file. The command is named a bit misleading because it just checks that the orm backend is not loaded and the orm module is not imported. These were two of the leading causes of slow down but not the exclusive ones. That's why I created the bash timing code. We can keep the `check-load-time` call I guess, doesn't hurt

Reply:
> `which verdi` will just resolve to the full path of the verdi binary, and it should then just be executed normally. (...)

Ah, I see the `` now in the `VERDI` parameter definition. Makes sense.